pkg/api/defaults-kubelet.go

```diff
@@ -315,6 +315,13 @@ func removeKubeletFlags(k map[string]string, v string) {
 			delete(k, key)
 		}
 	}
+
+	// Remove dockershim related flags in v1.24 and up
+	if common.IsKubernetesVersionGe(v, "1.24.0-alpha") {
+		for _, key := range []string{"--image-pull-progress-deadline", "--network-plugin"} {
+			delete(k, key)
```

Comment:
was going to ask if these should these removed or added back in if less than 1.24, but from above it looks like the pattern is to remove the flags

Reply:
Yea, I had the same question and was following previous examples.